server/routes/api/chart.py

```diff
@@ -541,12 +542,17 @@ def choropleth_data(dcid):
                 else:
                     geo_data[geo] = sv_data[geo][sv_date]
                     num_data_points += 1
+            exploreUrl = None
```

Comment:
could you just call:

exploreUrl = landing_page_api.build_url([dcid], [sv], bool(denom_data))